### DISCUSSION TITLE
Hitsound sounds distorted

### DISCUSSION THREAD

**Discussion by taep96** [ppy/osu#15863] [Q&A] [answered]
While music is untouched, sound that plays at the start of the song and all hitsounds are distorted. I don't know if I can properly describe it but it sounds like they have been amplified and reduced back in volume.

I have made a recording using a speaker:

https://user-images.githubusercontent.com/64481039/143938734-9e0a52fe-a988-4cb0-8d45-a6e226a39a38.mp4

My audio settings:
![image](https://user-images.githubusercontent.com/64481039/143936003-744289de-3b22-4f28-a827-69a77f7e255e.png)
OS: Pop!_OS
Audio library: PipeWire (not stock)

**Comment by bdach** [comment, marked as answer]
please review your audio configuration. i believe it's highly likely something in there is at fault with this, because i do not have any such problems on linux (i do not use pipewire).

**Reply by taep96** [reply]
I have reviewed it before submitting the issue. I also tried using all audio options osu! provides. It's either like I described it: normal audio with distorted hitsounds, or like I had it before switching to pipewire: silent (system wide) and ALL sound distorted in osu!.

**Reply by bdach** [reply]
to reiterate, i do not have your exact system configuration and cannot reproduce this issue. therefore this discussion is basically at a stalemate, and the issue will likely not be fixed until someone who has the requisite skills and a configuration wherein this reproduces puts in the effort to debug this.

as stated previously linux support is not an immediate priority anyhow.

**Comment by smoogipoo** [comment]
Have you tried using the "Default" option instead of the "PipeWire Sound Server" option?

**Reply by taep96** [reply]
Yes, I tried every option, https://github.com/ppy/osu/discussions/15863#discussioncomment-1718430
